pkg/repositories/config/migrations.go

```diff
@@ -1097,6 +1097,89 @@ var NoopMigrations = []*gormigrate.Migration{
 			return nil
 		},
 	},
+
+	{
+		ID: "2023-08-04-admin-tags",
+		Migrate: func(tx *gorm.DB) error {
+			type AdminTag struct {
+				gorm.Model
+				Name string `gorm:"index:,unique;size:255"`
+			}
+
+			return tx.AutoMigrate(&AdminTag{})
+		},
+		Rollback: func(tx *gorm.DB) error {
+			return nil
+		},
+	},
+
+	{
+		ID: "2023-08-04-noop-migration-execution-admin-tags",
```

Comment:
can we drop the name `noop` from this migration? This migration creates the join table, so maybe add that to the name or at the very least leave a comment.

Reply:
done